### DISCUSSION TITLE
Do you support .nuxtignore files?

### DISCUSSION THREAD

**Discussion by JangCool** [nuxt/nuxt#16160] [Questions]
I want to manage .vue files and .ts files with the same name in pages.

ex) login.vue, login.ts

The .ts file is recognized as a page(.vue) file and an error occurs when accessing the URI.

How can I exclude specific items from the pages folder?

![image](https://user-images.githubusercontent.com/2518021/142163937-2fb5abfb-42c1-4225-8776-a487d1e3624c.png)
![image](https://user-images.githubusercontent.com/2518021/142163827-b759ac49-4deb-4234-a179-9178db8eacf2.png)

**Comment by danielroe** [comment]
This should now be possible with https://github.com/nuxt/framework/pull/3424 - .nuxtignore` can be configured as in nuxt 2, for example, the below would exclude .ts files from your pages directory:

```
pages/**/*.ts
```

**Reply by hay-kot** [reply]
I've been trying to figure out how to do this for some time and always thought it was impossible. Would it make sense to add this example to the 'Pages' docs to show how you can colocate pages and their specific Javascript/Typescript? 

I'm up for making a PR to add an example!
